controllers/topics/index.js

```diff
@@ -29,6 +28,7 @@ const UserTopicService = require('../../services/postgres/UserTopicService');
 const getSubscribableTopic = require('./getSubscribeableTopic');
 const {getAnonymUser} = require('../../utils/getAnonymUser');
 const UsersFunction = require('../../databases/functions/users');
+const {insertTopics} = require('../../utils/post');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `insertTopics` is added here. Ensure that the function `insertTopics` exists in the path `'../../utils/post'` and it's exported correctly from there.



<!-- This is an auto-generated comment by OSS CodeRabbit -->